packages/grid/_modules_/grid/hooks/features/columns/useGridColumns.ts

```diff
@@ -36,57 +37,60 @@ import { useGridApiOptionHandler } from '../../root/useGridApiEventHandler';
 import { GRID_STRING_COL_DEF } from '../../../models/colDef/gridStringColDef';
 import { GridComponentProps } from '../../../GridComponentProps';
 
-function updateColumnsWidth(columns: GridColumns, viewportWidth: number): GridColumns {
-  const numberOfFluidColumns = columns.filter((column) => !!column.flex && !column.hide).length;
-  let flexDivider = 0;
-
-  if (numberOfFluidColumns && viewportWidth) {
-    columns.forEach((column) => {
-      if (!column.hide) {
-        if (!column.flex) {
-          viewportWidth -= Math.max(column.minWidth!, column.width!);
-        } else {
-          flexDivider += column.flex;
-        }
+type RawGridColumnsState = Omit<GridColumnsState, 'lookup'> & {
+  lookup: { [field: string]: GridColDef | GridStateColDef };
+};
+
+function getStateColumns(
```

Comment:
As far as I can see this handles only the `width` of the columns? Maybe we can have a more descriptive name, like `getComputedColumns` or `recomputeColumnsWidth` or something else?

Reply:
If you prefer the naming `computedColumns` over `stateColumns` I can rename `GridStateColDef` into `GridComputedColDef` and `getStateColumns` into `getComputedColumns`

I'm note a fan to limit the method name to the concept of `width`.

Reply:
My argument was because it only takes care of the `width` but as it is now is also ok.

Reply:
We can also argue that its best to keep a name related to the width for now and rename it to its generic naming when a 2nd preprocessing occurs.